packet_analysis/sql/dao/arp.py

```diff
@@ -0,0 +1,13 @@
+from scapy.all import Packet
+from ..dao.sqlObject import sqlObject
+from ..dao.ethernet import Ethernet, table_sql as ethernet_table_sql
+
+def table_sql() -> str:
```

Comment:
What's the point of this function if it just returns the output of another function?

Reply:
Organizationally it makes sense to be identical to all of the other objects.